python/pip_install/extract_wheels/lib/wheel.py

```diff
@@ -55,7 +55,10 @@ def entry_points(self) -> Dict[str, str]:
             # Calculate the location of the entry_points.txt file
             metadata = self.metadata
             name = "{}-{}".format(metadata.name.replace("-", "_"), metadata.version)
-            entry_points_path = os.path.join("{}.dist-info".format(name), "entry_points.txt")
+            # Note that the zipfile module always uses the forward slash as
+            # directory separator, even on Windows, so don't use os.path.join
+            # here.
```

Comment:
```suggestion
            # here.
            # Ref, for Python 3.10: https://github.com/python/cpython/blob/3.10/Lib/zipfile.py#L355
```

Substantiates the "always" claim.